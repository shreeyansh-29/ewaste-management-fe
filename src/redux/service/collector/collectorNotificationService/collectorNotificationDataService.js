import api from "../../../../core/utilities/httpProvider";
import {COLLECTOR_NOTIFICATION_MARKASREAD} from "../../../../container/constant/constant";

export const collectorNotificationDataService = () => {
  console.log("hy");
  return api.post(COLLECTOR_NOTIFICATION_MARKASREAD);
};
