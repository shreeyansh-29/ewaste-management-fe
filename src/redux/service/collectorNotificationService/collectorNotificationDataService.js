import api from "../../../core/utilities/httpProvider";
import * as types from "../../../container/constant/constant";

export const collectorNotificationDataService = () => {
  return api.post(types.COLLECTOR_NOTIFICATION_MARKASREAD);
};
