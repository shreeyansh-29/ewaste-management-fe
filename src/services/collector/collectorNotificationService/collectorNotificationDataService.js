import api from "../../../core/utilities/httpProvider";
import {COLLECTOR_NOTIFICATION_MARKASREAD} from "../../../container/constant/constants";

export const collectorNotificationDataService = () => {
  return api.post(COLLECTOR_NOTIFICATION_MARKASREAD);
};
