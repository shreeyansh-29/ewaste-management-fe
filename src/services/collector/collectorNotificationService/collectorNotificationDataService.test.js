/* eslint-disable no-undef */
import React from "react";
import Enzyme, {shallow} from "enzyme";
import {collectorNotificationDataService} from "./collectorNotificationDataService";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing collectorNotificationDataService", () => {
  const Service = collectorNotificationDataService;
  const wrapper = shallow(<Service />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
