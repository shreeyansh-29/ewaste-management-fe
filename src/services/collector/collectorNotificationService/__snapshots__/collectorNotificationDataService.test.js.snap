// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing collectorNotificationDataService 1`] = `<Component />`;
